src/Datadog.Trace.ServiceFabric/Remoting.cs

```diff
@@ -243,13 +243,28 @@ private static void InjectContext(PropagationContext context, IServiceRemotingRe
             }
         }
 
+        private static byte[] ConvertHexStringToByteArray(string hexString)
+        {
+            byte[] data = new byte[hexString.Length / 2];
+            for (var index = 0; index < data.Length; index++)
+            {
+                string byteValue = hexString.Substring(index * 2, 2);
+                data[index] = byte.Parse(byteValue, NumberStyles.HexNumber, CultureInfo.InvariantCulture);
+            }
+
+            return data;
+        }
```

Comment:
Can you try and compare the performance of https://stackoverflow.com/a/9995303/859275 ?

Reply:
Changed it to one from the link, and moved it to TraceId class.

Reply:
Resolve